wordpress-mobile/WordPress-iOS
# Edit Comment: add initial view

Ref: #17000 

This adds an initial new `Edit Comment` view. 
- It has sections for each editable field.
- The actual fields are empty and not editable yet.
- The nav bar `Cancel` and `Done` buttons simply dismiss the view for now.
- Display is controlled by the `newCommentEdit` feature flag, which is disabled by default.

To test:

---
With `newCommentEdit` enabled:
- Go to My Site > Comments > Comment details > Edit.
  - Verify the new edit view is displayed.
- Go to Notifications > Comments > Notification details > Edit. 
  - Verify the new edit view is displayed.

| iPhone | iPad |
|--------|-------|
| ![iphone](https://user-images.githubusercontent.com/1816888/129954831-e4f9693b-4d4b-4ff4-b010-3cbe0829a5e1.png) | ![ipad](https://user-images.githubusercontent.com/1816888/129954855-377631fb-80e6-47f9-a1ce-1a979776c319.png) |

---
With `newCommentEdit` disabled:
- Go to My Site > Comments > Comment details > Edit.
  - Verify the existing edit view is displayed.
- Go to Notifications > Comments > Notification details > Edit. 
  - Verify the existing edit view is displayed.

![edit_old](https://user-images.githubusercontent.com/1816888/129817182-e890dced-1024-4ff9-a6e9-66cab6d97461.png)

---

## Regression Notes
1. Potential unintended areas of impact
N/A. Feature is incomplete and disabled.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature is incomplete and disabled.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature is incomplete and disabled.

PR submission checklist:
